frontend/app/src/main/java/de/amosproj3/ziofa/bl/ConfigurationManager.kt

```diff
@@ -0,0 +1,84 @@
+// SPDX-FileCopyrightText: 2024 Luca Bretting <luca.bretting@fau.de>
+//
+// SPDX-License-Identifier: MIT
+
+package de.amosproj3.ziofa.bl
+
+import de.amosproj3.ziofa.api.ConfigurationAccess
+import de.amosproj3.ziofa.api.ConfigurationUpdate
+import de.amosproj3.ziofa.api.ProcessListAccess
+import de.amosproj3.ziofa.client.Client
+import de.amosproj3.ziofa.client.ClientFactory
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import uniffi.client.ClientException
+import uniffi.shared.Configuration
+import uniffi.shared.Process
+
+class ConfigurationManager(val clientFactory: ClientFactory) :
+    ProcessListAccess, ConfigurationAccess {
```

Comment:
Nice Job defining two clear interfaces for Listing and Configuration, thats what I would eventually like in our backend as well for testing.